packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-expandable-answer.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  buildSmartSnippet,
+  SmartSnippet,
+  SmartSnippetState,
+} from '@coveo/headless';
+import {h, Component, State, Prop, Element, Listen} from '@stencil/core';
+import ArrowDown from '../../images/arrow-down.svg';
+import {
+  InitializeBindings,
+  Bindings,
+  BindStateToController,
+} from '../../utils/initialization-utils';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-smart-snippet-expandable-answer',
```

Comment:
I'm unsure about the name of this component since there's also going to be an accordion of snippets in `atomic-smart-snippet-suggestions`. Its name in JSUI was `SmartSnippetHeightLimiter`.